From 5f690d29874d4283d07c185ffc0ad90942ab4814 Mon Sep 17 00:00:00 2001
From: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
Date: Thu, 4 Aug 2022 17:44:56 +0900
Subject: [PATCH 1/3] starterkit/config/mot: Change mot filepath

This patch changes to use same path both linux/windows.

Signed-off-by: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
---
 starterkit/config/mot | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/starterkit/config/mot b/starterkit/config/mot
index dfc2979..c8f9980 100644
--- a/starterkit/config/mot
+++ b/starterkit/config/mot
@@ -4,7 +4,7 @@
 ; But, M3 v1 board is too old, thus, it can't have CPLD 64bit mode.
 ; We use 32bit CPLD mode for all Starterkit boards.
 ;
-dir_mot	= "script\flash_writer\AArch32_output"
+dir_mot	= "starterkit"
 mot	= "AArch32_Flash_writer_SCIF_DUMMY_CERT_E6300400_ULCB.mot"
 
 num_cpld_cmd = 2
-- 
2.34.1

